### PR TITLE
Update included block example based on ones either side

### LINKED ISSUE #255: Better example for included blocks
I know it's my example, but like the fred/barney one, it's a specific use case rather than a good teaching tool. I'll assign myself to come up with an easier one to explain.

### PR DESCRIPTION
**Pull request by azaroth42**
Closes #255 ... finally!

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Apr 14, 2020, 6:07 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Frawcdn.githack.com%2Fw3c%2Fjson-ld-syntax%2F10a2770a5b4e8a217dea309daea69a634ba43505%2Findex.html%3FisPreview%3Dtrue)

```
Navigation timeout of 30000 ms exceeded
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/json-ld-syntax%23348.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8478,32 +8486,36 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
...
-        {"@id": "http://example.org/enum#s2"}
+      "http://example.org/team": [
+        {"@id": "http://example.org/leadership"}
       ]
     }, {
-      "@id": "http://example.org/base/3",
+      "@id": "http://example.org/person-2",
       "http://example.org/classification": [
-        {"@id": "http://example.org/enum#c6"}
+        {"@id": "http://example.org/contractor"}
```
Comment (iherman) [inline index.html:8518]: The statement on the name of Gregg is missing...

File: index.html [hunk 2]
```diff
@@ -8435,33 +8435,41 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
     "@context": {
       "@version": 1.1,
       "@vocab": "http://example.org/",
+      "schema": "http://schema.org/",
       "classification": {"@type": "@vocab"},
-      "service": {"@type": "@vocab"}
+      "team": {"@type": "@vocab"}
```
[the remark this answers is not in the document]
Comment (gkellogg) [reply index.html:8440]: Yes, we could settle on either "classification" or "team", don't need both.

File: index.html [hunk 3]
```diff
@@ -8478,32 +8486,36 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
...
     }, {
-      "@id": "http://example.org/base/2",
+      "@id": "http://example.org/leadership",
+      "@type": ["http://example.org/Team"],
+      "http://example.org/label": [{"@value": "Leadership Team"}]
+    }, {
+      "@id": "http://example.org/person-1",
       "http://example.org/classification": [
-        {"@id": "http://example.org/enum#c6"}
+        {"@id": "http://example.org/employee"}
```
[the remark this answers is not in the document]
Comment (gkellogg) [reply index.html:8510]: That could be told by running the "rake" task, which is also noted by travis. It checks that the first example matches the things in the second example after running through the algorithms.

```sh
bundle install
rake
```

That's all that should be necessary.

**Comment by azaroth42** [comment]
I wanted to demonstrate with the two different properties that it's not just a single relationship, it's a grab bag of any entities that you want to reference. If we don't think that's an issue, then yes, we can remove `team`.

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (iherman) [reply index.html:8440]: Actually, I was wondering about `"@type":"@value"`, not the fact of having both...

**Comment by gkellogg** [comment]
> I wanted to demonstrate with the two different properties that it's not just a single relationship, it's a grab bag of any entities that you want to reference. If we don't think that's an issue, then yes, we can remove `team`.

I don't see how this relates to included blocks; that's certainly what has been highlighted.

**Comment by azaroth42** [comment]
@BigBlueHat @pchampin ?

**Pull request merged**